jdbcSource/build.gradle

```diff
@@ -1,10 +1,14 @@
-group 'io.vantiq'
-version 'unspecified'
 
```

Comment:
Using new-style plugins closure requires moving it in gradle file.  No substantive changes here other than the new Lombok plugin